.dependabot/config.yml

```diff
@@ -6,4 +6,4 @@ update_configs:
     ignored_updates:
       - match:
           dependency_name: "postgres"
-          version_requirement: "10.7-alpine"
+          version_requirement: "11.8-alpine"
```

Comment:
This entire file can be remove now. It was only here to keep dependabot from trying to update postgres.